poseidon2/src/external.rs

```diff
@@ -179,6 +212,14 @@ impl<T, const WIDTH: usize> ExternalLayerConstants<T, WIDTH> {
         Self::new(initial_constants, terminal_constants)
     }
 
+    /// Construct constants from statically stored arrays, using a conversion function.
+    ///
+    /// This is useful when deserializing precomputed constants or embedding
+    /// them directly in the codebase (e.g., from `[[[u8; WIDTH]; N]; 2]` arrays).
```

Comment:
Nit: This will usually be u32 or u64.
```suggestion
    /// them directly in the codebase (e.g., from `[[[u32; WIDTH]; N]; 2]` arrays).
```